### DISCUSSION TITLE
Shared folder people can upload to without an account

### DISCUSSION THREAD

**Discussion by ClenonWolf** [filebrowser/filebrowser#2107] [Q&A]
I used to use filestash but I'm now switching to filebowser due to the way nicer looking and easier to use UI and amongst some other smaller features. Now filestash had a feature were you could share a folder in a mode where people with the link to the folder could upload/edit files in the folder. (https://www.filestash.app/img/screenshots/feature_share_0000.png)
Is there such a feature in filebrowser or could this be a future feature? Otherwise I might install both and only use filestash for shared folders as I heavily prefer filebrowser otherwise.

Thanks

**Comment by ramiresviana** [comment]
Filebrowser allows you to disable the authentication by using the `noauth` option.

**Comment by ediazrod** [comment]
Is not the same.. is only allow to some directory not at all.
